ios/FluentUI.Demo/FluentUI.Demo/Demos/SegmentedControlDemoController.swift

```diff
@@ -82,14 +82,51 @@ class SegmentedControlDemoController: DemoController {
         addPillControl(items: Array(segmentItems.prefix(2)),
                        style: .onBrandPill,
                        enabled: false)
+        container.addArrangedSubview(UIView())
+
+        addTitle(text: "Custom Colors")
+        addDescription(text: "not fixed width, unequal buttons", textAlignment: .center)
+        addPillControl(items: Array(segmentItems.prefix(4)),
+                       style: .primaryPill,
+                       equalSegments: false,
+                       isFixedWidth: false,
+                       customColors: true)
     }
 
     @objc func updateLabel(forControl control: SegmentedControl) {
         controlLabels[control]?.text = "\"\(segmentItems[control.selectedSegmentIndex].title)\" segment is selected"
     }
 
-    func addPillControl(items: [SegmentItem], style: SegmentedControlStyle, equalSegments: Bool = true, enabled: Bool = true, isFixedWidth: Bool = true) {
-        let pillControl = SegmentedControl(items: items, style: style)
+    func addPillControl(items: [SegmentItem], style: SegmentedControlStyle, equalSegments: Bool = true, enabled: Bool = true, isFixedWidth: Bool = true, customColors: Bool = false) {
+        let backgroundColor: UIColor?
+        let selectedBackgroundColor: UIColor?
+        let textColor: UIColor?
+        let selectedTextColor: UIColor?
+        let unreadDotColor: DynamicColor?
+        if customColors {
+            backgroundColor = UIColor(dynamicColor: DynamicColor(light: GlobalTokens.sharedColors(.bronze, .shade30),
+                                                                 dark: GlobalTokens.sharedColors(.bronze, .tint40)))
+            selectedBackgroundColor = UIColor(dynamicColor: DynamicColor(light: GlobalTokens.sharedColors(.steel, .tint40),
+                                                                         dark: GlobalTokens.sharedColors(.steel, .shade30)))
+            textColor = UIColor(dynamicColor: DynamicColor(light: GlobalTokens.sharedColors(.bronze, .tint40),
+                                                                 dark: GlobalTokens.sharedColors(.bronze, .shade30)))
+            selectedTextColor = UIColor(dynamicColor: DynamicColor(light: GlobalTokens.sharedColors(.steel, .shade30),
+                                                                   dark: GlobalTokens.sharedColors(.steel, .tint40)))
+            unreadDotColor = DynamicColor(light: GlobalTokens.sharedColors(.charcoal, .tint40),
+                                          dark: GlobalTokens.sharedColors(.charcoal, .shade30))
+        } else {
+            backgroundColor = nil
+            selectedBackgroundColor = nil
+            textColor = nil
+            selectedTextColor = nil
+            unreadDotColor = nil
+        }
+        let pillControl = SegmentedControl(items: items,
```

Comment:
Slight nit - this now skips the path of the standard init, reducing code coverage. Is there a way to re-write this to cover both initializers, so we have fewer opportunities for bugs to sneak in?